package.json

```
@@ -20,43 +20,43 @@
   "license": "MIT",
   "homepage": "https://github.com/maputnik/editor#readme",
   "dependencies": {
-    "@babel/runtime": "^7.1.2",
-    "@mapbox/mapbox-gl-rtl-text": "^0.2.2",
-    "@mapbox/mapbox-gl-style-spec": "^13.7.2",
+    "@babel/runtime": "^7.6.3",
+    "@mapbox/mapbox-gl-rtl-text": "^0.2.3",
+    "@mapbox/mapbox-gl-style-spec": "^13.9.0",
     "classnames": "^2.2.6",
-    "codemirror": "^5.40.2",
-    "color": "^3.0.0",
-    "detect-browser": "^4.5.0",
-    "file-saver": "^1.3.8",
+    "codemirror": "^5.49.0",
+    "color": "^3.1.2",
+    "detect-browser": "^4.7.0",
+    "file-saver": "^2.0.2",
     "jsonlint": "github:josdejong/jsonlint#85a19d7",
     "lodash.capitalize": "^4.2.1",
     "lodash.clamp": "^4.0.3",
     "lodash.clonedeep": "^4.5.0",
     "lodash.get": "^4.4.2",
     "lodash.isequal": "^4.5.0",
     "lodash.throttle": "^4.1.1",
-    "mapbox-gl": "^1.2.0",
+    "mapbox-gl": "^1.4.0",
```

```suggestion
    "mapbox-gl": "^1.4.1",
```

v1.4.1 is released which fixes a warning for images with empty string value.

---

Thanks @pathmapper change made